docs/Configuration.rst

```diff
@@ -4837,7 +4840,59 @@ database.
 -  The **EventCategoriesCache** size is approximately 24 bytes, so the
    maximum heap size equals to the cache size multiplied by 24 bytes.
    
-   
+
+Wallet caches
+~~~~~~~~~~~~~
+
+eXo Platform provides 2 wallet caches:
+
+-  :ref:`Wallet account cache <Wallet.accountCache>`
+
+-  :ref:`Wallet transaction cache <Wallet.transactionCache>`
+
+You can add new values of these caches in :ref:`exo.properties <Configuration.ConfigurationOverview>` file.
+
+.. code-block:: jproperties
+
+    exo.cache.wallet.account.MaxNodes=2000
+    exo.cache.wallet.account.TimeToLive=-1
+    exo.cache.wallet.transactions.MaxNodes=2000
+    exo.cache.wallet.transactions.TimeToLive=-1
+
+
+The specific configuration of wallet caches can be found in:
+
+-  ``$PLATFORM_TOMCAT_HOME/webapps/wallet-common.war!/WEB-INF/wallet/cache-configuration.xml``.
+
+.. _Wallet.transactionCache:   
+
+Wallet account cache
+-------------------------
+
+The **Wallet account cache** caches the Wallet objects.
+This object contains metadata information of a wallet, such as identity id, username, space pretty name, wallet enablement...
+When any user get access to his wallet or a space wallet, the cached wallet object will be retrieved from cache rather than the database.
```

Comment:
```suggestion
When any user get access to their wallet or a space wallet, the cached wallet object will be retrieved from cache rather than the database.
```

Reply:
Done